tutorial/notebook.ipynb

```diff
@@ -123,7 +123,7 @@
     "- [Ελληνικά](https://py-pdf.github.io/fpdf2/Tutorial-gr.html)\n",
     "- [עברית](https://py-pdf.github.io/fpdf2/Tutorial-he.html)\n",
     "- [Dutch](https://py-pdf.github.io/fpdf2/Tutorial-nl.html)\n",
-    "- [Polski]((https://py-pdf.github.io/fpdf2/Tutorial-pl.html))\n",
+    "- [Polski]((https://py-pdf.github.io/fpdf2/Tutorial-pl.html)\n",
```

Comment:
```suggestion
    "- [Polski](https://py-pdf.github.io/fpdf2/Tutorial-pl.html)\n",
```
Thnaks for spotting this problem!

I think that an extra starting parenthesis should also be removed